csi/moac/volume.js

```diff
@@ -34,12 +36,17 @@ class Volume {
     this.requiredNodes = _.clone(spec.requiredNodes || []).sort();
     this.requiredBytes = spec.requiredBytes;
     this.limitBytes = spec.limitBytes;
+    this.protocol = spec.protocol;
     this.size = 0;
     // state variables of the volume
     this.nexus = null;
     this.replicas = {}; // replicas indexed by node name
     this.state = 'pending';
     this.runFsa = 0; // number of requests to run FSA
+
+    if (status) {
+      this.size = status.size;
```

Comment:
it seems we don't have a test case where we would actually pass the size of the volume and test it is what it is expected to be.

Reply:
good catch I'll add it